Assets/Scripts/DetectLookedAtInteractive.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+
```

Comment:
Remove extra lines / whitespace unless you are systematically adding extra whitespace / line breaks to enhance the readability of your document (for example, by having a blank line separating distinct elements).